mensor/backends/sql.py

```diff
@@ -330,24 +330,25 @@ def _get_measures_sql(self, field_map, unit_type, measures, rebase_agg, stats_re
 
         if rebase_agg and stats:
             raise NotImplementedError("Computing stats and rebasing units simultaneously has not been implemented for the SQL backend.")
-        else:
-            for measure in measures:
-                if not measure.private:
-                    for fieldname, transforms in measure.get_fields(unit_type=unit_type, stats=stats, stats_registry=stats_registry, rebase_agg=rebase_agg).items():
-
-                        field = '1' if measure == 'count' else field_map['measures'][measure.via_name]
-                        if transforms.get('pre_agg'):
-                            field = transforms['pre_agg'](field, self.dialect)
-                        field = transforms['agg'](field, self.dialect)
-                        if transforms.get('post_agg'):
-                            field = transforms['post_agg'](field, self.dialect)
-
-                        aggs.append(
-                            '{col_op} AS {f}'.format(
-                                col_op=field,
-                                f=self._col(fieldname),
-                            )
-                        )
+
+        for measure in measures:
```

Comment:
this is just nit-y restructuring to reduce indent depth

Reply:
:+1: